packages/components/Link/src/Link.settings.ts

```diff
@@ -41,6 +47,13 @@ export const settings: IComposeSettings<ILinkType> = [
         tokens: {
           color: 'link'
         }
+      },
+      focused: {
+        content: {
+          style: {
+            borderColor: 'rgba(128, 128, 128, 1)',
```

Comment:
>128, 128, 128 [](start = 31, length = 13)

Will this work across different Office themes?

Reply:
@NikoAri I couldn't find the Office Theme color name for link focus. Do you know what it is?

Reply:
Please try contacting Alicia or Jason for theming related info

---
In reply to: [435718872](https://github.com/microsoft/fluentui-react-native/pull/241#discussion_r435718872) [](ancestors = 435718872)